assets/scss/pagination.scss

```diff
@@ -0,0 +1,207 @@
+.hale-page .moj-pagination__link:after,
+.hale-page .moj-pagination__link:before {
+  //correction for the font used on JotW.
+  position: relative;
+  top:-1px;
+}
+
+// MoJ Design System
+.moj-pagination {
+  // text-align: center;
+
+  @include govuk-media-query($from: desktop) {
+
+    // Alignment adjustments
+    margin-left: - govuk-spacing(1);
+    margin-right: - govuk-spacing(1);
+
+    // Hide whitespace between elements
+    font-size: 0;
+
+    // Trick to remove the need for floats
+    text-align: justify;
+
+    &:after {
+      content: '';
+      display: inline-block;
+      width: 100%;
+    }
+  }
+
+}
+
+.moj-pagination__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  @include govuk-media-query($from: desktop) {
+    display: inline-block;
+    margin-bottom: 0;
+    vertical-align: middle;
+  }
+}
+
+.moj-pagination__results {
+  @include govuk-font(19);
+  margin-top: 0;
+  @include govuk-media-query($from: desktop) {
+    display: inline-block;
+    margin-bottom: 0;
+    vertical-align: middle;
+  }
+}
+
+.moj-pagination__item {
+  @include govuk-font(19);
+  display: inline-block;
+}
+
+.moj-pagination__item--active,
+.moj-pagination__item--dots {
+  font-weight: bold;
+  height: 25px;
```

Comment:
Is it worth adding in a `rem` value too, for if people want to resize font-sizes in their browser? (We haven't tested if this works super well in Hale in general yet)